correlation_rules/github_advanced_security_change_not_followed_by_repo_archived.yml

```diff
@@ -1,23 +1,24 @@
 AnalysisType: correlation_rule
 RuleID: "GitHub.Advanced.Security.Change.NOT.FOLLOWED.BY.Repo.Archived"
```

Comment:
Maybe let's make the id `GitHub.Advanced.Security.Change.WITHOUT.Repo.Archived` to be consistent?

Reply:
Disregard my comment (We can't change RuleID.  It results in the old rule still being present in customer systems)